[2024-06-15T05:12:36.528+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dust_data_pipeline.ETL.populate_ultradust_summary_table scheduled__2024-06-15T04:00:00+00:00 [queued]>
[2024-06-15T05:12:36.549+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dust_data_pipeline.ETL.populate_ultradust_summary_table scheduled__2024-06-15T04:00:00+00:00 [queued]>
[2024-06-15T05:12:36.550+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-06-15T05:12:36.581+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): ETL.populate_ultradust_summary_table> on 2024-06-15 04:00:00+00:00
[2024-06-15T05:12:36.591+0000] {standard_task_runner.py:57} INFO - Started process 641 to run task
[2024-06-15T05:12:36.598+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dust_data_pipeline', 'ETL.populate_ultradust_summary_table', 'scheduled__2024-06-15T04:00:00+00:00', '--job-id', '274', '--raw', '--subdir', 'DAGS_FOLDER/test_redshift_real_v3.py', '--cfg-path', '/tmp/tmp1t_3rd1d']
[2024-06-15T05:12:36.600+0000] {standard_task_runner.py:85} INFO - Job 274: Subtask ETL.populate_ultradust_summary_table
[2024-06-15T05:12:36.691+0000] {task_command.py:410} INFO - Running <TaskInstance: dust_data_pipeline.ETL.populate_ultradust_summary_table scheduled__2024-06-15T04:00:00+00:00 [running]> on host 52eeab71c019
[2024-06-15T05:12:36.816+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dust_data_pipeline' AIRFLOW_CTX_TASK_ID='ETL.populate_ultradust_summary_table' AIRFLOW_CTX_EXECUTION_DATE='2024-06-15T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-15T04:00:00+00:00'
[2024-06-15T05:12:36.836+0000] {base.py:73} INFO - Using connection ID 'redshift_test_dev' for task execution.
[2024-06-15T05:12:36.912+0000] {sql.py:375} INFO - Running statement: 
    DROP TABLE IF EXISTS yusuyeon678.seoul_ultradust_summary;
    CREATE TABLE yusuyeon678.seoul_ultradust_summary (
        date TIMESTAMP primary key,
        ultradust FLOAT NOT NULL
    );
    , parameters: None
[2024-06-15T05:12:37.002+0000] {base.py:73} INFO - Using connection ID 'redshift_test_dev' for task execution.
[2024-06-15T05:12:37.073+0000] {sql.py:375} INFO - Running statement: 
    INSERT INTO yusuyeon678.seoul_ultradust_summary (date, ultradust)
    SELECT
        date,
        ROUND(AVG(ultradust), 2) AS ultradust
    FROM yusuyeon678.raw_data
    GROUP BY date;
    , parameters: None
[2024-06-15T05:12:37.175+0000] {sql.py:384} INFO - Rows affected: 15
[2024-06-15T05:12:37.269+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-06-15T05:12:37.282+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dust_data_pipeline, task_id=ETL.populate_ultradust_summary_table, execution_date=20240615T040000, start_date=20240615T051236, end_date=20240615T051237
[2024-06-15T05:12:37.332+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-06-15T05:12:37.358+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
